teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -141,7 +269,214 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowCallQueueOptOutChange
+
+When set to True users affected by the policy will be allowed to change the call queue whether to allow agent opt-out. When set to False (the default value) users affected by the policy will not be allowed to change the call queue whether to allow agent opt-out.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowCallQueueAgentOptChange
+
+When set to True users affected by the policy will be allowed to change the call queue's agent opt-in status. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's agent opt-in status.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowCallQueueMembershipChange
+
+When set to True users affected by the policy will be allowed to change the call queue's users. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's users.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowCallQueueRoutingMethodChange
+
+When set to True users affected by the policy will be allowed to change the call queue's routing method. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's routing method.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowCallQueuePresenceBasedRoutingChange
+
+When set to True users affected by the policy will be allowed to change the call queue's presence aware routing. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's presence aware routing.
```

Comment:
... allowed to change the call queue's presence-based routing option.
... not allowed to change the call queue's presence-based routing option.